Code/Tools/RemoteConsole/Core/RemoteConsoleCore.cpp

```diff
@@ -131,7 +131,10 @@ void SRemoteServer::StartServer()
 void SRemoteServer::StopServer()
 {
     m_bAcceptClients = false;
-    AZ::AzSock::CloseSocket(m_socket);
+    if (AZ::AzSock::IsAzSocketValid(m_socket))
```

Comment:
Related?

Reply:
No, just happens to error on startup because the RemoteConsole gets in here on init with an uninitialized socket. Just eliminating a red herring.

Reply:
👍